src/Couchbase.Lite.Shared/API/Database/Collection.cs

```diff
@@ -51,17 +87,33 @@ public sealed class Collection : IIndexable, IChangeObservable<CollectionChanged
         /// Cannot start with _ or %.
         /// Case sensitive.
         /// </remarks>
-        public string Name => throw new NotImplementedException();
+        public string Name { get; private set; } = DefaultCollectionName;
 
         /// <summary>
         /// Gets the Scope of the Collection belongs to
         /// </summary>
-        public Scope Scope => throw new NotImplementedException();
+        public Scope Scope { get; private set; }
```

Comment:
Is this set anywhere other than in the constructor?  If not, remove private set.